### DISCUSSION TITLE
akochan-reviewのエラーについて

### DISCUSSION THREAD

**Discussion by coins36** [majaneye/JANTAMA#7] [Q&A]
[エラー画像](https://imgur.com/a/ezgBVG3)
普段はこちらの[対処方法](https://bytemeta.vip/repo/Equim-chan/akochan-reviewer/issues/35)を参考にして解決しておりますが、JantamaViewerでakochan-reviewを使用する際は「set OMP_NUM_THREADS=8」を入力することができないため、エラーに対処できずにいます。どうすれば良いでしょうか。

**Comment by majaneye** [comment]
ご使用いただきありがとうございます！  
お手数をおかけして申し訳ございません。

OMP_NUM_THREADSはデフォルトでは設定されていない環境変数のようでした。  
なので、OMP_NUM_THREADSを追加することに関してはデメリットは無さそうです。（後で削除もできます）  

追加の方法は↓（気になる場合はあらかじめコマンドプロンプトを開いて、echo %OMP_NUM_THREADS%と入力してみてください。元々設定してある場合はその数が出ます。使用後に元の数で設定し直すこともできます）  
コマンドプロンプトを開いて（Windows10であれば、左下の［ここに入力して検索］にcmdと入力すれば開けます）、次のコマンドを入力してエンターキーを押す。
setx OMP_NUM_THREADS 8

コマンドプロンプトを閉じて、もう一度コマンドプロンプトを起動して、以下のコマンドを入力してエンターキーを押す。  
echo %OMP_NUM_THREADS%  

8と出力されれば、設定されています。  


参考：Windows10でTempやPathなどの環境変数を設定する方法  
https://proengineer.internous.co.jp/content/columnfeature/5205

**Reply by coins36** [reply]
詳しいご説明ありがとうございます。
おかげさまで問題解決できました！
